config/hooks/post-merge

```diff
@@ -4,7 +4,15 @@ red="$(tput setaf 1)"
 bold="$(tput bold)"
 reset="$(tput sgr0)"
 
+# Ensures up to data event schemas are downloaded
+mkdir -p schemas
+curl https://raw.githubusercontent.com/getsentry/sentry-data-schemas/main/relay/event.schema.json -o schemas/event.schema.json
+mypy snuba > /dev/null
+if [ $? -gt 1 ]; then
+  echo [${red}${bold}!!!${reset}] Could not dowload schemas.
+fi
```

Comment:
Isn't this just the same as the make step?

Reply:
Pretty much the same (the syntax is different because that one is in a Makefile). I added it in both because this happens automatically and you cannot really easily trigger it by yourself (technically you can execute the hook being it a shell script). The Makefile one is something you can always trigger manually.